src/LOAuth.ts

```diff
@@ -112,6 +112,42 @@ class LOAuth {
     return btoa(JSON.stringify(state));
   }
 
+  /**
+   * This method looks for a cookie set by PHC-Login that allows the
```

Comment:
I'm going to rework this to have this lib set the cookie as well, and use the cookie set method inside phc-login to avoid this disjointedness.